site/src/Librecores/SiteBundle/Resources/views/Default/home.html.twig

```diff
@@ -160,63 +160,107 @@
-      <a href="{{ path('librecores_planet_homepage') }}">Planet Librecores</a> collects blog posts from different
-      community members. Subscribe to it in your feed reader,
-      or <a href="{{ path('librecores_planet_homepage') }}">read it online</a>.</p>
+					<p>
+						<a class="btn btn-default"
+							href="{{ path('librecores_planet_homepage') }}">Read Planet
+							LibreCores &raquo;</a>
+					</p>
 
-      <p><a class="btn btn-default" href="{{ path('librecores_planet_homepage') }}">Read Planet LibreCores &raquo;</a></p>
-
-      {#
-      {% for post in blogposts %}
-          <p>
-            <div><a rel="nofollow" href="{{ post.url }}"><b>{{ post.author }}: {{ post.title }}</b></a></div>
-            <div>{{ post.teaser }}</div>
-          </p>
-      {% endfor %}
-      #}
-    </div>
-    <div class="col-sm-6">
-      <h2>Contribute to LibreCores</h2>
-      <p>LibreCores is a true community project. All development is done in the open, and we're always open to new contributors!
-      If you have only a couple of minutes to spare, or a full week, there's something for you to do.</p>
-      <p>Have a look at our contribution guide and get started today!</p>
-      <p><a class="btn btn-default" href="http://librecores-web.readthedocs.io/en/latest/contributing.html">Read on how to contribute to LibreCores &raquo;</a></p>
-    </div>
-  </div>
+					{# {% for post in blogposts %} <p> <div><a rel="nofollow" href="{{
+					post.url }}"><b>{{ post.author }}: {{ post.title }}</b></a></div>
+					<div>{{ post.teaser }}</div> </p> {% endfor %} #}
+				</div>
+			</div>
+			<div class="row">
+				<div class="col-sm-12">
+					<h2>Contribute to LibreCores</h2>
+					<p>LibreCores is a true community project. All development is
+						done in the open, and we're always open to new contributors! If
+						you have only a couple of minutes to spare, or a full week,
+						there's something for you to do.</p>
+					<p>Have a look at our contribution guide and get started today!</p>
+					<p>
+						<a class="btn btn-default"
+							href="http://librecores-web.readthedocs.io/en/latest/contributing.html">Read
+							on how to contribute to LibreCores &raquo;</a>
+					</p>
+				</div>
+			</div>
+		</div>
+		<div class="col-sm-6">
+			<h2>Recent Activity</h2>
+			{% for project in activity %}
+			<p>
+				<strong><a
+					href="{{ path('librecores_project_repo_project_view', {'parentName':
+          project.getParentName(), 'projectName': project.getName()}) }}">{{ project.getFqname() }}</a></strong>
+				was
+				{% if project.getDateLastModified() == project.getDateAdded() %}
+				added
+				{% else %}
+				updated
+				{% endif %}
+				<time
+					datetime="{{ project.getDateLastModified()|date("c", "UTC") }}"
+					data-livestamp="{{ project.getDateLastModified()|date("c", "UTC") }}">
+					on
+					{{ project.getDateLastModified()|date("d F Y H:i", "UTC") }}
+					UTC
+				</time>
+				.
+			</p>
+			{% endfor %}
+		</div>
+	</div>
```

Comment:
Please get rid of the tabs in this file as well. HTML is usually indented with two spaces in our current code. And then rebase/squash so we can take that as a single commit. Thanks!